p2pclient/p2pclient.go

```diff
@@ -26,15 +27,23 @@ type Client struct {
 
 	mhandlers sync.Mutex
 	handlers  map[string]StreamHandlerFunc
+
+	openPersistentConn sync.Once
+	// persistent connection writer and reader
+	pConnWriter ggio.WriteCloser
```

Comment:
```suggestion
	persistentConnWriter ggio.WriteCloser
```

The comment is (1) not synchronized (there is no reader here) and (2) can be replaced with the proper variable name.